docat/docat/models.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
```

Comment:
A general question also for @fliiiix and maybe @timofurrer :

As the `...Response` models are now also internally used and not only for JSON Responses I would
argue we should remove the `Response` suffix. What do you think?